packages/riipen-ui/src/components/Tab.jsx

```diff
@@ -108,7 +108,7 @@ const Tab = props => {
           font-weight: ${theme.typography.fontWeight.regular};
           font-size: 14px;
           letter-spacing: 2px;
-          line-height: 1.4;
+          line-height: 20px;
```

Comment:
ditto possibility of using theme.typography.<variant>.lineHeight here if possible